components/Common/Search/index.tsx

```diff
@@ -27,6 +28,13 @@ export const SearchButton: FC = () => {
     }
   });
 
+  const { os } = useDetectOS();
+
+  let osCommandKey = 'Ctrl';
+  if (os === 'MAC') {
+    osCommandKey = '⌘';
+  }
```

Comment:
```suggestion
  const osCommandKey = os === 'MAC' ?  '⌘' : 'Ctrl';
```

you should do that, also don't forget to run `npx turbo format`.

Reply:
Sure. And I already ran `npx turbo format`. Thanks for the review. 
P.S.: updated the font, since there are only two variants and `ibm-plex-mono` has a very similar result as the default `kbd` element (`monospace`), I went for `open-sans`.